src/wwwroot/app/views/vacancy/vacancy.edit.controller.js

```diff
@@ -166,4 +170,17 @@ export default function VacancyController( //eslint-disable-line max-statements
    function _getStateTitle(key) {
       return $translate.instant(key);
    }
+
+// The first location of the vacancy should match the responsible user's location
+   function _vacacyEditValidation(vacancy) {
```

Comment:
it seems not correct naming of function. it is too general. more concrete name can be better, for example - vacancyLocationValidation or some else.